R/execute.R

```diff
@@ -110,10 +110,12 @@ execute_array_udf <- function(namespace, array, udf, layout=NULL, selectedRanges
   # object.
   if (typeof(resultObject) != "raw") {
     className <- class(resultObject)[1]
+    print(resultObject)
+    print(str(resultObject))
```

Comment:
Calling `str()` strikes me as a bad idea; it is a somewhat 'internal' view.  Users may or may not be R programmers.

Reply:
@eddelbuettel  good call; these were debug-prints which needn't stay